test/test_parabolic_2d.jl

```diff
@@ -124,16 +128,28 @@ end
                                  "elixir_navierstokes_convergence.jl"),
                         cells_per_dimension=(4, 4), tspan=(0.0, 0.1),
                         l2=[
-                            0.0015355076812512347,
-                            0.0033843168272690953,
-                            0.0036531858107444093,
-                            0.009948436427520873
+                            0.0015355076237431118,
+                            0.003384316785885901,
+                            0.0036531858026850757,
+                            0.009948436101649498
                         ],
                         linf=[
-                            0.005522560467186466,
-                            0.013425258500736614,
-                            0.013962115643462503,
-                            0.0274831021205042
+                            0.005522560543588462,
+                            0.013425258431728926,
+                            0.013962115936715924,
+                            0.027483099961148838
+                            #= FIXME: Values from above (PR) or values from below (main)?
```

Comment:
With the values from main these tests fail, right (without adjusting the tolerances)? By "Values from above (PR)" do you mean the values CI obtains or the ones you get locally?

Reply:
I think this message was automatically generated by Github actions, so I'm not sure what it means. I'm guessing `main` introduced new `l2, linf` values?

Reply:
Removed the comments.